syndeo-template/src/test/java/com/google/cloud/syndeo/it/PubsubDlqIT.java

```diff
@@ -243,6 +243,60 @@ public void testPubsubToPubsubFailure() throws Exception {
             .build());
   }
 
+  @Test
+  public void testPubsubToPubsubLargeMsgFailure() throws Exception {
+    // Start Syndeo pipeline
+    FlexTemplateClient templateClient =
+        FlexTemplateClient.builder().setCredentials(CREDENTIALS).build();
+    PipelineOperator operator = new PipelineOperator(templateClient);
+
+    PipelineLauncher.LaunchInfo syndeoPipeline = kickstartSyndeoPipeline();
+
+    // Publish large messages to the source queue
+    publishLargeMsgToPubsub();
+
+    // Wait for the Syndeo pipeline to move all the data.
+    Thread.sleep(5 * ONE_MINUTE_MILLIS);
```

Comment:
Is this necessary? Can we use something like http://www.awaitility.org/ to make sure we check every X seconds instead of hardcoding this time?